nest_elephant_tvb/translation/FatEndPoint.py

```diff
@@ -0,0 +1,196 @@
+from mpi4py import MPI
+import pathlib
+
+def make_connections(path_to_files_receive, path_to_files_send, logger_master):
+    '''
+    Fat end point, here: connections to nest and tvb.
+    '''
+    use_mpi = True
+    # init MPI
+    if use_mpi:
+        comm = MPI.COMM_WORLD  # INTRA communicator
+        info = MPI.INFO_NULL
+        root=0
+    else:
+        comm = None
+    ### nest connection
```

Comment:
Spurious comment: repeats the name of the function call

Reply:
Added the comment for better/quicker readability of the code by others. Can be removed if seen not necessary.